### DISCUSSION TITLE
DOWNLOAD PROBLEMS

### DISCUSSION THREAD

**Discussion by jc47004** [rickloader/NewForestRouteV3#24] [General]
Greetings.   Having problems downloading the files.....   GitHub showing errors. and ZIP is saying Network issues.....  Any thoughts.  Is it possible to receive the files on a CD/DVD Disk??    Many thanks.   John Coles    zzjc@btinternet.com
PS. I'll try the ZIP at a different time of day.  possibly BUSY Network?

**Comment by sweiland-openrails** [comment]
You could use OR unstable version. Menu has a "Content" button. When pressed you get a form with which the route can be downloaded (next to others). Highlight the route and press "Install". It also has a "Update" button with which updates for the route can be installed.

Downloading the route via zip fails all the times. I've seen it, others too.

regards, Siebren.

**Comment by jc47004** [comment]
Do you mean "content" on the OR program, or on the OR website.   Cannot seem to find.
OTHERS.....Found CONTENT on the OR program.

**Comment by jc47004** [comment]
Have successfully downloaded Demo Mode 1 & ME Coast from OR website (zipped)
But UNABLE to download Chiltern & New Forest.....(either via ZIP or GitHub.)
Any thoughts.... Thanks JOHN   zzjc@btinternet.com

**Comment by jc47004** [comment]
Hurrah....  Finally managed to d/load the Chiltern ZIP via GitHub....  Have extracted zip (took about one hour)  Will now try launching it!!
Cheers...  Got there in the end....

**Comment by rickloader** [comment]
Any thoughts ? Well as you ask,  I can`t understand why you ignore my advice not to download the zip because this is often faulty. Desktop is preferable. I don`t mean to be nasty about this. I hope it works for you.
I think it is significant that both New Forest and chiltern are much bigger than  ME or demo routes
________________________________

**Comment by rickloader** [comment]
There are 2 reliable ways to download NF Route and  .zip is not one of them. Don`t use it! The files downloaded will be incomplete.
The recommended download is either
1) Install Github Desktop and use this to download and update NF Route
https://desktop.github.com/download/
OR
2) Run the OpenRails Unstable version "content" page .Download from within Open Rails  (As Siebren suggests above) The Open Rails Devs have arranged that this download will be the most recent GitHub version.

The route is work in progress. It is in continuous development so it is important to download and update using the methods above
Rick

**Reply by jc47004** [reply]
Have now downloaded New Forest (NF) via Github.     Thanks for all your help....

**Comment by jc47004** [comment]
Have successfully downloaded New Forest via Github. 
1st attempt got to around 90% then failed
But 2nd try was fine....     Thanks for your help.


Sent from Mail for Windows
